apps/frontend/src/libs/components/button/button.tsx

```diff
@@ -1,11 +1,26 @@
 type Properties = {
+	className: string;
+	clickBtn?: (event_: React.BaseSyntheticEvent) => void;
```

Comment:
Callbacks passed to props should be named with on prefix, like onClick. ( rule A9 of quality criteria)

Reply:
+1, this is a better suggestion than mine

Reply:
thank you, that's make improvement for me